src/views/Player.vue

```diff
@@ -262,13 +262,13 @@ export default {
             let plioQuestion = {
```

Comment:
This is a very strange loop -- hadn't noticed this before. You can replace it with a `questions.forEach` loop and it will look cleaner, I think....

Maybe do it off in this PR only

Reply:
Yep, will do! 👍